### PR TITLE
SCIM docs - index and changelog

### PR DESCRIPTION
**Pull request by anle-statsig**

### CODE REVIEW

**Comment by anle-statsig** [comment]
* **#2128** <a href="https://app.graphite.dev/github/pr/statsig-io/docs/2128?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @anle-statsig and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by jasonwzm** [comment]
🚀 Deployed on https://67005478aa1f281cd066a434--cozy-fox-0defec.netlify.app

File: docs/client/concepts/parameter-stores.mdx [hunk 1]
```diff
@@ -32,5 +32,5 @@ If you've had this problem already, you'll recognize the power immediately.  If
 Parameter Stores are available in:
 - Android SDK v4.33.0+
 - iOS SDK v1.45.0+
-- @statsig/js-client and @statisg/react-bindings v1.4.0+
+- @statsig/js-client and @statsig/react-bindings v1.4.0+
```
Comment (anle-statsig) [inline docs/client/concepts/parameter-stores.mdx:35]: @tore-statsig is this a typo or intended?

**Comment by anle-statsig** [comment]
### Merge activity

* **Oct 4, 4:52 PM EDT**: @anle-statsig started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/statsig-io/docs/2128).

File: docs/client/concepts/parameter-stores.mdx (hunk 1 above)
Comment (tore-statsig) [reply docs/client/concepts/parameter-stores.mdx:35]: nice fix, thanks

**Pull request merged**